spec/models/spotlight/blacklight_configuration_spec.rb

```diff
@@ -450,4 +450,40 @@
       end
     end
   end
+
+  describe 'document_index_view_types=' do
+    it 'Array should pass through' do
+      array = [1, 2, 3]
+      subject.document_index_view_types = array
+      expect(subject.document_index_view_types).to match_array array
+    end
+
+    it 'Hash should be simplified to Array' do
+      checkboxes_from_form = { 'list' => '1', 'gallery' => '1', 'map' => '0' }
+      subject.document_index_view_types = checkboxes_from_form
+      expect(subject.document_index_view_types).to match_array %w(list gallery)
+    end
+  end
+
+  describe 'document_index_view_types_selected_hash' do
```

Comment:
Can we rename `document_index_view_types_selected_hash` to `#document_index_view_types_selected_hash` for consistency with rspec naming conventions?